x-pack/platform/plugins/shared/alerting/server/application/rule/methods/bulk_edit/bulk_edit_rules.ts

```diff
@@ -545,6 +545,7 @@ async function updateRuleAttributesAndParamsInMemory<Params extends RuleParams>(
       references,
       params: updatedParams,
       actions: actionsWithRefs,
+      artifacts: artifactsWithRefs,
     } = await extractReferences(
```

Comment:
@cnasikas I haven't done any change to the `bulk_edit_rule` method, until CI notified me that `extractReferences` method is being used here as well and thus I had to make the change. This left me with some uncertainty that other methods might be possibly broken. Could that be the case? Can we solely rely on CI (type checking and and current code coverage) that there won't be any other breakings in other methods? I rely on your team on manual testing the other methods if you think it is necessary.